Appz/Appz/Apps/WhatsApp.swift

```diff
@@ -41,6 +42,15 @@ extension Applications.WhatsApp.Action: ExternalApplicationAction {
                 ),
                 web: Path()
             )
+        case .Send(let abid, let text):
+            return ActionPaths(
+                app: Path(
+                    pathComponents: ["send"],
+                    queryParameters: ["text" : text, "abid": abid ?? ""]
```

Comment:
You don't need to use null coalescing `??` here, unless `abid` is an optional string `String?`.